scss/_patterns_tabs.scss

```diff
@@ -97,3 +160,19 @@
     }
   }
 }
+
+@mixin vf-p-tabs-light-theme {
+  @include vf-p-tabs-theme(
+    $colors--text-default: $colors--light-theme--text-default,
```

Comment:
The mixin variable naming seems a bit inconsistent. Parameters should just name what color refers to, so I'd suggest `$color-tabs-text`, `$color-tabs-border`, `$color-tabs-highlight-hover`.